### PR TITLE
Feature/nav bar

### LINKED ISSUE #9: As a User, I should ALWAYS have access to a tap bar at the bottom of the screen for navigation
The tap bar should have three buttons that will be different depending on the page view

### LINKED ISSUE #10: The tap bar navigation should change depending on what path I am viewing
On '/' I should see, from left to right: an X, a User, a <3

On '/favorites' I should only see a home(house icon) in the center where the User was. I should not see the X or the <3

### LINKED ISSUE #11: As a user, if I like the city I am viewing, I should be able to press the <3 button and add to my favorites
I should be able to press this button at any time (I shouldn't have to scroll to the bottom of the page to press it).

### LINKED ISSUE #15: When I press the user icon the url should change to /favorites
I should see an h2 with "Your Favorites"

### LINKED ISSUE #16: As a user I should see a message if I don't have any favorites
If i haven't added any favorites yet, I should see a message below "My favorites" that says "You don't have any yet, click here to get searching!" with an arrow pointing to the home button

### PR DESCRIPTION
**Pull request by clairefields15**
# Description of Changes/Motivation
Added a nav bar to the bottom of the page. I assume most of my users will be on mobile, but need to add media queries to bring this to the top for desktop.
The nav bar allows navigation between the home page and a user's favorite page. Relevant buttons are hidden/shown and the url changes.
You can click on the heart to add a city to your favorites. Not sure if I love the modal that's happening now, I might add buttons to close it instead of set timeout or try to find some other visual indicator that a city has been added to your favorites.

Need to test all of this still.

## Issues Closed
closes #9 
closes #15 
closes #10
closes #16 
closes #11 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -55,39 +59,77 @@ export const App = () => {
...
+      setShowModal(false);
+      window.scrollTo(0, 0);
     }
   }, [cityImage]);
 
+  const addToFavorites = async e => {
+    const duplicate = favorites.find(favorite => favorite.name === cityName);
+
+    if (!duplicate) {
+      await showModalTimeout(1000);
```
Comment (AshleyOh-bit) [inline src/components/App/App.js:71]: Good compromise for the UX on this one. Now your users won't miss that they've added a favorite!

File: src/components/Nav/Nav.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { useLocation, NavLink } from 'react-router-dom';
+import './Nav.css';
+import home from '../../images/home.png';
+import nope from '../../images/nope.png';
+import likeBlack from '../../images/likeBlack.png';
+// add a hover state to the like button?
```
Comment (AshleyOh-bit) [inline src/components/Nav/Nav.js:7]: Kayla told Lourdes and I to try and avoid saving comments onto the main branch, and rather to make sure they are saved in a previous version of the file. Totally up to you, of course!

File: cypress/integration/nav_and_favorite_spec.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  });
+
+  it('Should see a message if I have no favorites', () => {
+    cy.get('a').contains('Favorites').click();
+    cy.contains('Your Favorites');
+    cy.contains("You don't have any favorites yet");
+  });
+
+  it('Should be able to add a city to my favorites list', () => {
+    cy.get('[id=like-btn]').click();
```
Comment (AshleyOh-bit) [inline cypress/integration/nav_and_favorite_spec.js:68]: What happens if you click "Nope" from this view?

File: src/components/App/App.js [hunk 4]
```diff
@@ -55,39 +59,77 @@ export const App = () => {
...
       <ScrollToTop />
-      <Logo />
+      {!showModal && <Logo />}
+      <Nav
+        addToFavorites={addToFavorites}
+        favorites={favorites}
+        cityName={cityName}
+      />
       {!errorMessage && isLoading && <h2>Loading...</h2>}
       {!!errorMessage && !isLoading && (
```
Comment (AshleyOh-bit) [inline src/components/App/App.js:101]: Back at it with the double bang operator, I see.

**Review by AshleyOh-bit — APPROVED** [review]
Great job! I had trouble finding much to comment on. Some of the CSS could be drier, but not by much. I would just make sure to test the Nope flow with Cypress!

File: src/components/App/App.js (hunk 4 above)
Comment (clairefields15) [reply src/components/App/App.js:101]: Always

File: cypress/integration/nav_and_favorite_spec.js (hunk 3 above)
Comment (clairefields15) [reply cypress/integration/nav_and_favorite_spec.js:68]: Currently literally nothing haha working on it!

**Pull request merged**